index.html

```diff
@@ -15,16 +15,17 @@
   <main class="wrapper">
     <header class="main-header">
       <nav class="navbar">
-        <div id="open-menu" class="display-flex-column"><i class="fa fa-bars fa-xl" aria-hidden="true"></i></div>
-        <ul id="menu-items" class="flex-list">
-          <li class="navbar-links display-flex-column"><a href="#">Hello</a></li>
-          <li class="navbar-links display-flex-column"><a href="#">Portfolio</a></li>
-          <li class="navbar-links display-flex-column"><a href="#">About</a></li>
-          <li class="navbar-links display-flex-column"><a href="#">Contact</a></li>
+        <div class="open-menu display-flex-column"><i class="fa fa-bars fa-xl" aria-hidden="true"></i></div>
+        <ul class="menu-items flex-list">
+          <li class="navbar-links display-flex-column"><a href="#home">Hello</a></li>
+          <li class="navbar-links display-flex-column"><a href="#projects">Portfolio</a></li>
+          <li class="navbar-links display-flex-column"><a href="#about">About</a></li>
```

Comment:
* Kindly use the right color for the mobile menu list as given in the Figma design 
## Figma design ![image](https://user-images.githubusercontent.com/52497246/223379507-d46e3e3c-01e2-4467-b806-38be66f3bab1.png)
## Yours 
![image](https://user-images.githubusercontent.com/52497246/223379940-b4556d5e-613b-46d7-a146-6f93464fb84d.png)